Fix plugin docker mount documentation (#3124)
neo4j/docs-operations: modules/ROOT/pages/docker/plugins.adoc

n10s
----

> If you rely on the `NEO4J_PLUGINS` environment variable, ensure you map explicit host permissions to avoid container folder ownership blocks and override the server directory mappings inside your environmental configuration:

Author: ```suggestion
If you rely on the `NEO4J_PLUGINS` environment variable, override the default server directory mappings inside your environmental configuration:
```

[source, shell, subs="attributes"]
----
# Add to your container environment variables
NEO4J_server_directories_plugins=/plugins
----

> 

Reviewer: again, no this is not necessary. It is done for the user, even when using NEO4J_PLUGINS.

> NEO4J_server_directories_plugins: /plugins

Author: ```suggestion
NEO4J_server_directories_plugins=/plugins
```
